src/utils/getOriginalFileForArchiveRecord.ts

```diff
@@ -0,0 +1,15 @@
+import { DerivativeType } from '@permanentorg/sdk';
```

Comment:
Same situation here -- this is a "new file" because so many lines changed, but the only actual change should be variable names.